### PR TITLE
[enhance](mtmv)support replace materialized view

### PR DESCRIPTION
**Pull request by zddr**
- support `alter MATERIALIZED VIEW mv1 replace with  MATERIALIZED VIEW mv2 PROPERTIES('swap' = 'false');`
- fix rename MATERIALIZED VIEW  not change state of high level MATERIALIZED VIEW

Logic is the same as: https://doris.apache.org/zh-CN/docs/dev/sql-manual/sql-statements/Data-Definition-Statements/Alter/ALTER-TABLE-REPLACE/ 

The difference is that this statement can only operate on materialized views

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run p0

File: regression-test/suites/mtmv_p0/test_replace_mtmv.groovy [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+    test {
+          sql """
+              alter MATERIALIZED VIEW ${tableName1} replace with  MATERIALIZED VIEW ${mvName1};
+          """
+          exception "MATERIALIZED_VIEW"
+    }
+
+    sql """
+        alter MATERIALIZED VIEW ${mvName1} replace with  MATERIALIZED VIEW ${mvName2};
```
Comment (zfr9527) [inline regression-test/suites/mtmv_p0/test_replace_mtmv.groovy:110]: Will an exception occur when this statement is executed while mtmv is being updated?

**Review by zfr9527 — APPROVED** [review]
LTGM

File: regression-test/suites/mtmv_p0/test_replace_mtmv.groovy (hunk 1 above)
Comment (zddr) [reply regression-test/suites/mtmv_p0/test_replace_mtmv.groovy:110]: no,will waiting for the lock in very little time

**Review by morrySnow — APPROVED** [review]

**Pull request merged**